a8cteam51/sift-decisions
# Add event tests

#### Changes proposed in this Pull Request

* Adds a test and fixes some deprecated function calls

#### Testing instructions



* Start environment: `npm i && composer install && npx wp-env start`
* Tests included: `npm test`
* Manually you can run it by going to <http://localhost:8888/?post_type=product> and adding a product to the cart. (There's not really a way to check it was called yet -- probably need to make a log statement or something.)

Mentions https://github.com/Automattic/gold/issues/502
